src/components/calendar/bl-calendar.css

```diff
@@ -19,7 +19,7 @@
   justify-content: space-between;
   width: 100%;
   align-items: center;
-  gap: 2px;
+  padding-bottom: 15px;
```

Comment:
Should we use variables instead of giving a fixed width?
https://baklava.design/?path=/docs/design-system-size-and-spacing--documentation